das/metrics.go

```diff
@@ -0,0 +1,132 @@
+package das
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/instrument/asyncint64"
+	"go.opentelemetry.io/otel/metric/instrument/syncfloat64"
+	"go.opentelemetry.io/otel/metric/instrument/syncint64"
+
+	"github.com/celestiaorg/celestia-node/header"
+)
+
+var (
+	meter = global.MeterProvider().Meter("das")
+)
+
+type metrics struct {
+	sampled    syncint64.Counter
+	sampleTime syncfloat64.Histogram
+
+	newHead syncint64.Counter
+
+	busyWorkers        asyncint64.Gauge
+	isRunning          asyncint64.Gauge
+	headHeight         asyncint64.Gauge
+	headOfSampledChain asyncint64.Gauge
+}
+
+func (sc *samplingCoordinator) initMetrics() error {
+	sampled, err := meter.SyncInt64().Counter("das_sampled_headers_count",
+		instrument.WithDescription("sampled headers counter"))
```

Comment:
```suggestion
		instrument.WithDescription("total count of sampled headers"))
```

Reply:
Thank you!.  This one meant to be **not** a total count, but I struggled to come up with better naming. 
Explanation: This one is tricky, because technically counter is not a total sum, it could do counts over duplicates and will increase upon retries. So counter value could be higher than actual amount of sampled headers.

Reply:
Ahhh, I see, makes sense. I would just leave it then, I also cannot think of a better name